profitbricks-rest/src/test/java/org/apache/jclouds/profitbricks/rest/binder/volume/CreateSnapshotRequestBinderTest.java

```diff
@@ -34,7 +41,14 @@
    @Test
    public void testCreatePayload() {
       
-      Injector injector = Guice.createInjector(new GsonModule());
+      Injector injector = Guice.createInjector(new GsonModule(), new AbstractModule() {
+         @Override protected void configure() {
+            bind(new TypeLiteral<Supplier<URI>>() {
+            }).annotatedWith(Provider.class).toInstance(
+               Suppliers.ofInstance(URI.create(new ProfitBricksProviderMetadata().getEndpoint())));
+            }
+         });
```

Comment:
This will be needed in all binder tests. Create a base test class for all binder tests that initializes the injector variable in a `@BeforeClass` method, so tests can just use it and focus on tests.